atd-vzd/migrations/migration_crashes_2020_03_07--0221.sql

```diff
@@ -0,0 +1,7 @@
+-- We need to drop this index Hasura will not see
+drop index idx_atd_txdot_crashes_crash_id;
+
+-- And add a primary key constraint that will be detected by Hasura
+alter table atd_txdot_crashes
+	add constraint atd_txdot_crashes_pkey
+		primary key (crash_id);
```

Comment:
The index for crashes needed removal, and a new constraint needed for Hasura to detect.